docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -889,15 +890,44 @@ testcafe chrome my-tests --sf
 
 ### --ts-config-path \<path\>
 
-Enables TestCafe to use a custom [TypeScript configuration file](../guides/concepts/typescript-and-coffeescript.md#customize-compiler-options) and specifies its location.
+Deprecated as of [TestCafe v.X.X.X] in favour of the --compiler-options parameter.
```

Comment:
Add the preferred usage examples since the `compiler-options` option is complex. Maybe your can split this example and add appropriate parts into descriptions for each interface (command-line, programmatic, configuration file).

```js
*Command-line interface*

testcafe chrome tests --compiler-options typescript.configPath=<path-to-custom-ts-config.json>

*Programmatic interface*
runner.compilerOptions(
     {
         'typescript': { configPath: 'path-to-custom-ts-config.json'}
     }
)

*Configuration file*
{
    "compilerOptions": {
         "typescript": { configPath: "path-to-custom-ts-config.json"}
    }
}
```